File: C prog/day3/prime.c
#include <stdio.h>

int main()
{
    int num, i = 2, count = 0;

    printf("Enter a number: ");
    scanf("%d", &num);

    while (i < num)
    {
        if (num % i == 0)
        {
            count++;
        }
        i++;
    }

    if (num <= 1 || count > 0)
        printf("%d is not a prime number.\n", num);
    else
        printf("%d is a prime number.\n", num);

    return 0;
}

File: eclipse basic java/training/dsa/src/loops/Prime.java
package loops;

public class Prime {
	public static void main(String[] args) {
		int n = 50, num = 2;        
		while(num <= n) { 
			int i = 2; 
			boolean prime = true; 

			while(i <= num / 2) {   
				if(num % i == 0) {                 
					prime = false;                  
					break; 
			}       
				i++; 
			} 

			if(prime) { 
				System.out.print(num + " "); 
			} 
			num++; 
		} 

	}
}
